Config/AltBuildSystems/TasmanianSG.py

```diff
@@ -1134,6 +1204,12 @@ def isWavelet(self):
         '''
         return (self.pLibTSG.tsgIsWavelet(self.pGrid) != 0)
 
+    def isFourier(self):
+        '''
+        returns True if using a Fourier grid
+        '''
+        return (self.pLibTSG.tsgIsFourier(self.pGrid) != 0)
```

Comment:
* nothing wrong, just giving a comment
* in C programming language, when integers are used as booleans, e.g.,
```
int a;
...
if (a) ...
```
then `0` is interpreted as `false`, while any non-zero value is `1`. It is better to make the check explicit, but always stick to the convention:
```
if (a != 0)
```
every time integers are used as flags for booleans.